tools/rum-models-generator/Sources/RUMModelsGeneratorCore/Output/Printing/SwiftPrinter.swift

```diff
@@ -6,20 +6,12 @@
 
 import Foundation
 
-/// A type which prints Swift code.
-public protocol SwiftCodePrinter {
-    func print(swiftTypes: [SwiftType]) throws -> String
-}
-
 /// Generates Swift code from `SwiftTypes`.
-public class SwiftPrinter: Printer, SwiftCodePrinter {
-    // MARK: - Printing
-
+public class SwiftPrinter: BasePrinter {
     public func print(swiftTypes: [SwiftType]) throws -> String {
         reset()
 
         try swiftTypes.forEach { type in
-            precondition(indentationLevel == 0)
```

Comment:
What's the reason behind removing this check?

Reply:
It was quite handy for development, but as now the generated Swift syntax is asserted in `SwiftPrinterTests` I thought this `precondition()` is unnecessary.